cs/src/json/JSON.csproj

```diff
@@ -19,16 +26,22 @@
     <Compile Include="protocols\SimpleJsonReader.cs" />
     <Compile Include="protocols\SimpleJsonWriter.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(BuildFramework)' == 'net45-nonportable'">
```

Comment:
Result is still the same. The behavior of the solution is less confusing with the current mechanism.